src/Item_Project.php

```diff
@@ -265,16 +270,100 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (
+                    Session::haveRight("project", Project::READMY)
+                    && $item instanceof CommonDBTM
+                    && in_array($item->getType(), $CFG_GLPI["project_asset_types"])
+                ) {
+                    self::showForAsset($item);
+                }
+                break;
         }
         return true;
     }
+
+    private static function showForAsset(CommonDBTM $item): void
+    {
+
+        $item_project = new self();
+        $item_projects = $item_project->find([
+            'itemtype' => $item->getType(),
+            'items_id' => $item->getID()
+        ]);
+
+        $used = $entries = [];
+
+        foreach ($item_projects as $value) {
+            $used[] = $value['projects_id'];
+            $project = new Project();
+            $result = $project->getFromDB($value['projects_id']);
+
+            if ($result === false || !$project->can($project->fields['id'], READ)) {
```

Comment:
If you ignore completely the projects that cannot be viewed by the user. It will result in a list of items that has less items that the number shown in the tab counter.

In this case, we often limit viewed information to the name itself (`getLink()` adapts its result to the current user rights), and we display empty data in the other fields.